### DISCUSSION TITLE
[newbie] add user

### DISCUSSION THREAD

**Discussion by mgiammarco** [ltsp/ltsp#247] [Q&A]
Hello,
I am trying LTSP 19. I have done the simplest configuration without chroot.
Now I add an user with adduser. I boot a thinclient but I am not able to login, the password is not accepted. I can login on server with that user.
In addition: how can I use ldap users?
Thanks,
Mario

**Comment by alkisg** [comment]
Hi, in LTSP 20, after adding users on the server, you need to run `ltsp initrd` and reboot the clients.
This is documented in the installation page.

Regarding LDAP, search the ltsp issues, the ltsp community issues, and the wiki, it's been discussed a number of times.

**Comment by Android23051983** [comment]
Hello!
I try to create users but the system swears /usr/share/ltsp/client/login/pamltsp:181/u no further visible
What to do? Write me on email raki48@mail.ru

**Comment by alkisg** [comment]
This was because sshfs wasn't installed in the VM image.
